packages/web/src/store/selectors/scanDisplayOptions/packageSorters.ts

```diff
@@ -15,6 +15,14 @@ export const compareSeverities = (severityA = 'UNKNOWN', severityB = 'UNKNOWN')
 type PackageSorterComparator = (pkgA: IdentifiedPackage, pkgB: IdentifiedPackage) => number;
 
 const packageSorterMap: Record<PackageSortType, PackageSorterComparator> = {
+  name: (pkgA, pkgB) => pkgB.name.localeCompare(pkgA.name),
+  size: (pkgA, pkgB) => {
+    if (pkgA?.approximateByteSize && pkgB?.approximateByteSize) {
```

Comment:
We still want to sort things in cases like `pkA` has size, but `pkgB` doesn't. I'd suggest rewriting this as
`size: (pkgA, pkgB) => (pkgA?.approximateByteSize ?? 0) - (pkgB?.approximateByteSize ?? 0)`